deploy/crds/toolchain_v1alpha1_usersignup_crd.yaml

```diff
@@ -71,11 +71,7 @@ spec:
                 (“sub” claim)
               type: string
             username:
-              description: The username.  This may differ from the UserSignup's metadata.name,
-                which is restricted by the limited character set available for naming
-                (see RFC1123).  If the username contains characters which are disqualified
-                from the resource name, the username is transformed into an acceptable
-                resource name instead. For example, johnsmith@redhat.com -> johnsmith-at-redhat-com
+              description: The user's username, obtained from the identity provider.
```

Comment:
it's missing the `compliantUsername` parameter

Reply:
The script updated this file also.. not sure why it missed the new field.  I'll have to look at it tomorrow, it's past 10pm here now.

Reply:
because it's not in the correct format - see my comment here: https://github.com/codeready-toolchain/api/pull/45#discussion_r333973362

> it's past 10pm here now.

I know - I was surprised that you sent a PR at this time. Have a good night and weekend ;-)

Reply:
Now fixed.